cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/release_test.go

```diff
@@ -708,6 +778,24 @@ func TestCreateInstalledPackage(t *testing.T) {
 			if err := mock.ExpectationsWereMet(); err != nil {
 				t.Errorf("there were unfulfilled expectations: %s", err)
 			}
+
+			// check expected HelmReleass CRD has been created
+			dynamicClient, _, err = s.clientGetter(context.Background())
+			if err != nil {
+				t.Fatalf("%+v", err)
+			}
```

Comment:
Excellent, thanks for adding the checks for the release CRD in the unit tests.

Reply:
you got it